user_and_channel_report

```diff
@@ -12,6 +13,8 @@ import slack_channel_report
 import report_generator
 import report_targets
 
+roy = "U06NSQT34"
```

Comment:
Can you move this into a file that's not committed to git?

Reply:
Yeah, just submitted a change.